### PR TITLE
update poc interval to 300

### PR DESCRIPTION
**Pull request by cokes518**

### CODE REVIEW

**Review by danielcolinjames — APPROVED** [review]
👍  sorry, I will get this automated soon 😅

**Comment by allenan** [comment]
can we have this just pull from the chain vars?

**Comment by cokes518** [comment]
That’s the dream

File: components/Widgets/ChecklistWidget.js [hunk 1]
```diff
@@ -21,24 +22,26 @@ const ChecklistWidget = ({ hotspot, witnesses, isDataOnly }) => {
       setActivityLoading(true)
       const hotspotActivity = await getActivityForChecklist(hotspotid)
       setActivity(hotspotActivity)
+      const {
+        data: { poc_challenge_interval },
+      } = await fetch('https://api.helium.io/v1/vars').then((res) => res.json())
```
Comment (allenan) [inline components/Widgets/ChecklistWidget.js]: we have vars in helium-js https://github.com/helium/helium-js/blob/master/packages/http/src/resources/Vars.ts

Comment (danielcolinjames) [reply components/Widgets/ChecklistWidget.js]: ah shoot, should've checked

Comment (allenan) [reply components/Widgets/ChecklistWidget.js]: there's also a singular client in `data/client` to make it easier to make http calls

**Pull request merged**